common/app/views/fragments/email/stylesheets/connectedStyleOverride.scala.html

```diff
@@ -29,7 +29,8 @@
     .tone-news .headline {
         color: #333333;
     }
-    .tone-news .fc-item__kicker {
+    .tone-news .fc-item__kicker,
+    .tone-news .byline {
```

Comment:
yep it's on my radar, I noticed there's a way to use sass for inline style blocks. out of interest has this been done for AMP yet? I think last time I looked AMP had raw inlined CSS like this